ql-qlf-plugin/synth_quicklogic.cc

```diff
@@ -192,7 +192,10 @@ struct SynthQuickLogicPass : public ScriptPass {
             std::string readVelArgs;
             readVelArgs = " +/quicklogic/" + family + "_cells_sim.v";
 
-            run("read_verilog -lib -specify +/quicklogic/cells_sim.v" + readVelArgs);
+            // Use -nomem2reg here to prevent Yosys from complaining about
+            // some block ram cell models. After all the only part of the cells
+            // library required here is cell port definitions plus specify blocks.
+            run("read_verilog -lib -specify -nomem2reg +/quicklogic/cells_sim.v" + readVelArgs);
```

Comment:
wouldn't using nomem2reg stop using registers for small memories?

Reply:
This option here does not affect the design. It is used only for loading cell definitions to prevent Yosys from infering RAMs in their simulation models.